src/styles/popover-inline.css

```diff
@@ -0,0 +1,142 @@
+/**
+ * Styles overrides for inline popover
+ */
+.ce-popover--inline {
+  --height: 38px;
+  --height-mobile: 46px;
+
+  position: relative;
+
+  .ce-popover__custom-content {
+    margin-bottom: 0;
+  }
+
+  .ce-popover__items {
+    display: flex;
+  }
+  
+  .ce-popover__container {
+    flex-direction: row;
+    padding: 4px;
+    height: var(--height);
+    top: 0;
+    
+    min-width: max-content;
+    width: max-content;
+    animation: none;
+
+    @media (--mobile) {
+      height: var(--height-mobile);
+      position: absolute;
+    }
+  }
+
+  /** 
+   * Popover item styles
+   */
+  .ce-popover-item-separator {
+    padding: 0 4px;
+
+    &__line {
+      height: 100%;
+      width: 1px;
+    }
+  }
+
+  .ce-popover-item {
+    border-radius: 4px;
+    padding: 4px;
+
+    &__icon--tool {
+      box-shadow: none;
+      background: transparent;
+      margin-right: 0;
+    }
+    
+    &__icon {
+      width: unset;
+      height: unset;
+
+      svg {
+        width: var(--icon-size);
+        height: var(--icon-size);
+
+        @media (--mobile) {
+          width: var(--icon-size--mobile);
+          height: var(--icon-size--mobile);
+        }
+      }
+    }
+
+    &:not(:last-of-type) {
+      margin-bottom: unset;
+    }
+  }
+
+  .ce-popover-item-html {
+    display: flex;
+    align-items: center;
+  }
+
+  .ce-popover-item__icon--chevron-right {
+    transform: rotate(90deg);
+  }
+
+  .ce-popover--nested-level-1 {
```

Comment:
nested popover in inline popover should not have left offset

<img width="533" alt="image" src="https://github.com/codex-team/editor.js/assets/3684889/71f3e727-bf39-4cc7-b48e-f84883ed3532">

Reply:
fixed